### PR TITLE
update cookbook_gallery yaml with Radar cookbook

### PR DESCRIPTION
**Pull request by jukent**
The Project Pythia Cookbooks gallery will only display internal cookbooks correct? External ones are links in the Resource gallery. So I've updated the yaml to have a link to @mgrover1 's Radar Cookbook for accuracy in representing the direction we are headed.

### CODE REVIEW

File: portal/cookbook_gallery.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-- title: CMIP6
-  url: http://gallery.pangeo.io/repos/pangeo-gallery/cmip6/
+- title: Radar Cookbook
+  url: https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html
   description: |
-    Examples of analysis of Google Cloud CMIP6 data using Pangeo tools.
+    This Project Pythia Cookbook covers the basics of working with weather radar data in Python using Py-ART.
```
Comment (mgrover1) [inline portal/cookbook_gallery.yaml:4]: Can you remove the mention of Py-ART here? Although the current content only covers that material, we want to make this flexible enough to add other Python modules (Ex. wradlab) in the future

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion - other than that, this looks great!

**Review by mgrover1 — APPROVED** [review]

**Comment by mgrover1** [comment]
Thanks @jukent for making that change!! 😁

File: portal/cookbook_gallery.yaml (hunk 1 above)
Comment (jukent) [reply portal/cookbook_gallery.yaml]: Done! I was a little curious about that. I've also added tags in this push.

**Review by ktyle — APPROVED** [review]

**Pull request merged**